{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/css/wysiwyg/wysiwyg.css

```diff
@@ -11,6 +11,14 @@
 
 /* We are styling root elements so we need access to type */
 /* stylelint-disable selector-no-type */
+p {
+  margin-bottom: 1vr;
+}
+
+ul {
+  list-style: disc;
+}
+
```

Comment:
As they're removed by the `@include Reset_List()`, would it be a good idea to restore browser default padding/margins?